influxql/iterator.gen.go

```diff
@@ -418,6 +418,7 @@ type floatFillIterator struct {
 	curTime    int64
 	startTime  int64
 	endTime    int64
+	auxFields  []interface{}
```

Comment:
Auxiliary fields are fields attached to a point. For example in the query:

```
SELECT min(value), fieldA, fieldB FROM cpu
```

The point represents `min(value)` but the auxiliary fields are `fieldA` and `fieldB` which are the values at that particular data point.

There's some write up of the query engine in the godocs but I'll write up a more substantial overview in an `influxql` README or godoc.